File: c-sharp/Controllers/CaesarController.cs
using System;
using System.Collections.Generic;
using System.Text;
using Ciphers.Models;

namespace Ciphers.Controllers
{
    public class CaesarController : BaseController
    {
        public CaesarText GetCaesarText(CaesarText cText)
        {
            int shiftVal = cText.Key;
            foreach (char plnChar in cText.PlainText)
            {
                cText.StrBuild.Append(CipherService.GetCipherLetter(plnChar, shiftVal));
            }

            cText.CipherText = cText.StrBuild.ToString();
            return cText;
        }
    }
}

File: java/src/main/java/com/metatarsal/ciphers/controllers/CaesarController.java
package com.metatarsal.ciphers.controllers;

import com.metatarsal.ciphers.models.CaesarText;
import org.springframework.stereotype.Controller;

@Controller
public class CaesarController extends BaseController {

    public CaesarText caesarCipher(CaesarText text) {

        StringBuilder sb = getStringBuilder();

        for (int i = 0, n = text.getPlainText().length(); i < n; i++) {
            sb.append(cipherService.getCipherLetter(text.getPlainText().charAt(i),
                                                    text.getKey()));
        }

        text.setCipherText(sb.toString());

        return text;
    }
}
